### DISCUSSION TITLE
Request for additional overclock options

### DISCUSSION THREAD

**Discussion by goodfodder** [eduardofilo/RG350_adam_image#104] [General]
Deeply appreciate additional oc options as the two on offer crash my original rg350 where 1080 was stable. Don’t see much value in so many under clocking options given the weak cpu

Many Thanks

**Comment by eduardofilo** [comment]
The overclocking options are determined by the capabilities of the chipset and by the configurations that are made in the kernel, that is, it cannot be set to an arbitrary speed. In ODbeta specifically the only possible options are:
444, 492, 600, 696, 792, 900, 996,1092, 1200

At the end of the file `/media/data/local/home/.simplemenu/config.ini` there is the next configuration:

```
[CPU]
normal_speed = 996
overclocked_speed_low = 1092
overclocked_speed_high = 1200
```

So in SimpleMenu we are playing with the higher speeds possible on ODbeta.

1080 is a speed that is possible in ROGUE, but not in ODbeta. It is a pity, because in my two JZ4770 machines, it works perfectly, but the close 1092 doesn't works in neither of the two. But it is what it is.
